tests/ts_cluster_keys.erl

```diff
@@ -0,0 +1,169 @@
+-module(ts_cluster_keys).
```

Comment:
The key changes seem flexible now but in the feature we will think of them just as keys.